common/utils.py

```diff
@@ -265,9 +267,23 @@ def convert_zip_file_to_tar_bytes(file_dir, filename):
     return tar_bytes
 
 
-def compare_string_element_in_list(existing_list, new_list):
-    added = []
-    removed = []
-    updated = []
-
-    return added, removed, updated
+def send_email_notification(recipients, notification_subject, notification_message, notification_arn, boto_util):
+    for recipient in recipients:
```

Comment:
Why cant we send the notification to all the recipients by keeping them in to list or CC.